ebos/eclmpiserializer.hh

```diff
@@ -90,9 +90,11 @@ public:
         if constexpr (is_ptr<T>::value) {
             ptr(data);
         } else if constexpr (is_pair<T>::value) {
-            pair(data);
+            tuple(data);
```

Comment:
Maybe we should add a specialization of `is_tuple` for pair and use that instead of `is_pair` above.

Reply:
done in previous pr.